apps/site/lib/site_web/controllers/schedule/trip_info.ex

```diff
@@ -102,6 +102,8 @@ defmodule SiteWeb.ScheduleController.TripInfo do
         origin_id: conn.query_params["origin"],
         destination_id: conn.query_params["destination"]
       )
+    else
+      error -> error
```

Comment:
It looks to me like you want to return `{:error, error} `instead of only `error` here so it might get picked up in case statement in `handle_trip/3`.

Reply:
Wait, unless `error` is already that tuple.. 🤯 I don't know.

Reply:
You can handle both if needed:
```elixir
else
  {:error, _} = error -> error
  error -> error
end
```

Reply:
> Wait, unless `error` is already that tuple.. 🤯 I don't know.

Yes, exactly, the error is already a tuple :) I will add a test though.